.codeclimate.yml

```diff
@@ -16,7 +16,6 @@ plugins:
         go:
         java:
         javascript:
-        php:
```

Comment:
Do we need to remove this? I think this is a standard file we include in every repo.